dspace-api/src/main/java/org/dspace/content/service/FeedbackService.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content.service;
+import java.io.IOException;
+import javax.mail.MessagingException;
+import javax.servlet.http.HttpServletRequest;
+
+import org.dspace.core.Context;
+
+/**
+ * Service interface class for the Feedback object.
+ * 
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk@4science.com)
+ */
+public interface FeedbackService {
+
+    public void sendEmail(Context context, HttpServletRequest request, String recipientEmail, String senderEmail,
```

Comment:
Please add JavaDocs to this method to describe what it does and what the parameters are